### PR TITLE
343 static colors

### PR DESCRIPTION
**Pull request by neodyymi**
RoutePath colors now come from a static known list of colors.

### CODE REVIEW

File: src/util/colorScale.ts [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
+        '#ffe119',
+        '#4363d8',
+        '#f58231',
+        '#42d4f4',
+        '#f032e6',
+        '#fabebe',
+        '#469990',
+        '#e6beff',
+        '#9A6324',
+        '#fffac8',
```
Comment (ahjyrkia) [inline src/util/colorScale.ts]: #fffac8 quite hard to see on the map. Maybe darken it a bit?

File: src/util/colorScale.ts [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
-        return chroma.scale(['red', 'yellow', 'green', 'blue'])
-        .mode('lch').colors(colorCount);
+    public reserveColor() {
+        if (this.colorStack.length < 1) {
+            return '#007ac9';
+        }
+        return this.colorStack.pop();
+    }
+
+    public releseColor(color: string) {
```
[the remark this answers is not in the document]
Comment (ahjyrkia) [reply src/util/colorScale.ts]: releaseColor*

File: src/util/colorScale.ts (hunk 1 above)
Comment (neodyymi) [reply src/util/colorScale.ts]: Good point! I think we just drop it from the list..

File: src/util/colorScale.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (neodyymi) [reply src/util/colorScale.ts]: Should be fixed now.

**Pull request merged**